packages/loot-core/migrations/1691233396000_add_schedule_next_date_tombstone.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE schedules_next_date ADD COLUMN tombstone INTEGER DEFAULT 0;
```

Comment:
This fixes another (different) issue.

Reproduction:
1. create a new schedule
2. ctrl+z to undo the creation
3. before: it breaks; now: it works